provision/testdata/flavor_cko_calico-3.23.2.inp.yaml

```diff
@@ -23,6 +23,28 @@ aci_config:
     external_networks:
     - mykube_extepg
 
+connectivity_checker:
+ error_pods_reporting_enable: true
+ reachability_test_enable:
+  periodic_sync:
+   enable_updates: true
+   interval: 120
+  external_url:
+   url: 'google.com'
+  proxy: ""
```

Comment:
We dont need to add another proxy field. The "cko_proxy_config" section already has the option to set the proxy. Please use the same field.

Reply:
addressed